Week1/homework/app.js

```diff
@@ -1,11 +1,50 @@
 'use strict';
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
-  ];
 
+const booksTitles = {
+  'titles': ['gone_with_the_wind', 'the_lord_of_the_rings', 'the_other_woman', 'war_and_peace', 'song_of_solomon', 'the_fountainhead', 'the_shadow_of_the_wind', 'the_golden_compass', 'catch_22', 'the_kite_runner'],
+  'languages': ['english', 'english', 'english', 'english', 'english', 'english', 'english', 'english', 'english', 'english'],
+  'authors': ['author1', 'author2', 'author3', 'author4', 'author5', 'author6', 'author7', 'author8', 'author9', 'author10'],
+  'img': ['0.jpg', '1.jpg', '2.jpg', '3.jpg', '4.jpg', '5.jpg', '6.jpg', '7.jpg', '8.jpg', '9.jpg']
+};
 
-  // Replace with your own code
-  console.log(bookTitles);
+const booksList = document.createElement("section");
+booksList.setAttribute("id", "booksList");
+
+const ul = document.createElement("ul");
+
+document.body.appendChild(booksList);
+booksList.appendChild(ul);
+
+for (let i = 0; i < Object.keys(booksTitles.titles).length; i++) {
+  if (
+    (booksTitles.img[i],
+    booksTitles.languages[i],
+    booksTitles.authors[i] === undefined)
+  ) {
```

Comment:
this if statement is not working; in conditional statements no commas, I highly recomend for you this [video](https://www.youtube.com/watch?v=FzNQ2USorGY) or similer to it named similler to `truthy vs falsy values js`.